src/components/proofs/List.tsx

```diff
@@ -1,31 +1,25 @@
-import { AccentText } from 'components/Text'
-import { Suspense } from 'preact/compat'
-import { space } from 'classnames/tailwind'
-import { useSnapshot } from 'valtio'
-import ERC721ProofsList from 'components/proofs/ERC721ProofsList'
-import EmailProofList from 'components/proofs/EmailProofList'
+import { Suspense, SuspenseList } from 'preact/compat'
+import CardTitle from 'components/CardTitle'
 import ListTitle from 'components/proofs/ListTitle'
-import LoadingCard from 'components/proofs/LoadingCard'
-import Scrollbar from 'components/Scrollbar'
-import proofStore from 'stores/ProofStore'
+import LoadingCard from 'components/LoadingCard'
+import ProofList from 'components/proofs/ProofList'
 
 export default function () {
-  const { proofsCompleted } = useSnapshot(proofStore)
-
   return (
-    <Suspense fallback={<LoadingCard />}>
-      <ListTitle />
-      <Scrollbar>
-        <div className={space('space-y-2')}>
-          <ERC721ProofsList />
-          <EmailProofList />
-        </div>
-      </Scrollbar>
-      {proofsCompleted.length > 0 && (
-        <AccentText small primary color="text-primary">
-          Created ZK proofs are saved in the browser even if you switch wallets.
-        </AccentText>
-      )}
-    </Suspense>
+    <SuspenseList revealOrder="together">
+      <Suspense
+        fallback={
+          <CardTitle
+            title="Loading..."
+            subtitle="Please, wait until I load supported NFTs, it takes time"
+          />
+        }
+      >
+        <ListTitle />
+      </Suspense>
+      <Suspense fallback={<LoadingCard title="" subtitle="" />}>
```

Comment:
Why do you need to pass `title` and `subtitle` here, if they're optinal in the component